### PR TITLE
update video grant

### PR DESCRIPTION
**Pull request by vassjozsef**

### CODE REVIEW

File: lib/twilio-ruby/jwt/access_token.rb [hunk 1]
```diff
@@ -69,6 +69,11 @@ def payload
           payload
         end
 
+        class << self
```
Comment (efossier) [inline lib/twilio-ruby/jwt/access_token.rb]: Isnt the ConversationsGrant different from the VideoGrant? Not sure we want to be modifying it.

**Comment by efossier** [comment]
Since this will be published in a new RC I think we can skip deprecating the old configuration_profile_sid since this is clearly marked as alpha software.

File: lib/twilio-ruby/jwt/access_token.rb (hunk 1 above)
Comment (vassjozsef) [inline lib/twilio-ruby/jwt/access_token.rb]: Removed.

**Comment by vassjozsef** [comment]
ConversationsGrant removed.

**Comment by efossier** [comment]
+1

**Pull request merged**